### PR TITLE
Nazar/feature/refacroting v1

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

File: backend/app/router/order.py [hunk 1]
```diff
@@ -19,9 +19,7 @@ def get_orders(
 
     log(log.INFO, "get_orders, business_id: [%d]", business.id)
 
-    return s.OrdersOut(
-        orders=db.query(m.Order).filter_by(business=business, is_deleted=False).all()
-    )
+    return s.OrdersOut(orders=db.query(m.Order).filter_by(business=business).all())
```
Comment (Taraslysak) [inline backend/app/router/order.py:22]: `return return s.OrdersOut(orders=business.orders.all())` ?

File: backend/app/router/phone_number.py [hunk 2]
```diff
@@ -21,12 +21,18 @@ def create_check_phone_number(data: s.CreatePhoneNumber, db: Session = Depends(g
...
+    number = data.phone_number
 
-    phone_number = db.query(m.PhoneNumber).filter_by(number=data.phone_number).first()
+    is_number_valid(number)
+
+    phone_number = (
+        db.query(m.PhoneNumber)
+        .filter(m.PhoneNumber.number.ilike(f"%{number}%"))
+        .first()
+    )
```
Comment (Taraslysak) [inline backend/app/router/phone_number.py:32]: Why cant we search for exact phone match here?

File: frontend/src/pages/Market/components/PersonalInfo/PersonalInfo.tsx [hunk 3]
```diff
@@ -114,18 +118,25 @@ const PersonalInfo = ({
...
+                  return true;
                 }
-                return isValidPhoneNumber(v);
+                return false;
               },
             },
+            onChange: (e) => {
+              const targetValue = phoneNumberAutoFormat(e.target.value);
+              setValue(inputPhoneNumber, targetValue);
+            },
```
Comment (Taraslysak) [inline frontend/src/pages/Market/components/PersonalInfo/PersonalInfo.tsx:139]: No inline callbacks please

**Review by Taraslysak — CHANGES_REQUESTED** [review]

File: backend/app/router/order.py (hunk 1 above)
Comment (Nazar-kob) [reply backend/app/router/order.py:22]: Yes, the client's wish was to return all orders, even deleted ones

File: backend/app/router/phone_number.py (hunk 2 above)
Comment (Nazar-kob) [reply backend/app/router/phone_number.py:32]: Sometimes the number can consist of 10 digits or 12 as you use here ilike

File: frontend/src/pages/Market/components/PersonalInfo/PersonalInfo.tsx (hunk 3 above)
Comment (Nazar-kob) [reply frontend/src/pages/Market/components/PersonalInfo/PersonalInfo.tsx:139]: thanks done

**Review by kipod — APPROVED** [review]
10x

**Pull request merged**